pkg/defaults/policies/files/curl-cve.json

```diff
@@ -0,0 +1,36 @@
+{
+  "id": "db79755e-6e03-40f5-b914-b0bee9e1c20b",
+  "name": "Curl Buffer Overflow Vulnerabilities",
+  "description": "Alert on deployments with images containing severe curl and libcurl vulnerabilities (CVE-2023-38545 and CVE-2023-38546). There are flaws in curl command line tool and libcurl library in versions from 7.69.0 to 8.3.0.",
+  "rationale": "The vulnerability in CVE-2023-38545 makes curl overflow a heap based buffer in the SOCKS5 proxy handshake. The vulnerability in CVE-2023-38546) allows an attacker to insert cookies at will into a running program using libcurl.",
+  "remediation": "Upgrade curl or libcurl to version 8.4.0 or apply the patch available on curl's official website. If you are unable to update, avoid using CURLPROXY_SOCKS5_HOSTNAME proxies and do not set a proxy environment variable to socks5h://",
```

Comment:
1. Suggestion: `Upgrade curl or libcurl to version 8.4.0` -> `Upgrade curl and libcurl to version 8.4.0 or newer`
2. `apply the patch available on curl's official website` - patches seem to be for the source code. I wouldn't recommend this to an average user.